src/styles/blocks/card.scss

```diff
@@ -0,0 +1,94 @@
+$mainColor: #060b35;
+$secondaryColor: #616070;
+$blueAccent: #00acdc;
+
```

Comment:
move variables to separate file _variables.scss and https://github.com/import it in main.scss